### DISCUSSION TITLE
記事一覧で各記事のタイトルに付記されている日付について

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#421] [要望・アイデア・不具合]
記事一覧で、各記事のタイトルに付記されている日付は、

+ 検索結果の記事一覧では投稿日
+ マイページの記事一覧では更新日

となっています。
しかし、どちらかに統一されていないと色々と迷いが生じます。統一されていないそれなりの理由があればお聞かせください。

**Comment by wataru86** [comment]
@tsubolabo 
ご報告ありがとうございます🙏
検索システムに保存しているデータの都合上、検索ページに表示される日付が他ページと異なっていることが判明しました。
確かに現在の表記だと分かりづらいので、他の開発優先度なども踏まえて対応を検討します🙇
